app/views/projects/_latest_download_link.html.erb

```diff
@@ -1 +1,3 @@
-<%= link_to("Download latest report - generated #{time_ago_in_words(@latest_completed_download.completion_time)} ago", project_file_list_download_path(job_id: @latest_completed_download.job_id, target: "_blank")) %>
\ No newline at end of file
+<% if @latest_completed_download.completion_time %>
```

Comment:
I seem to have one "completed" job without a completion time on my local environment. This handles that weird case. Not sure how that record got there but I just added the check so that it does not blow up on the user if it ever happens in prod.